src/main/app/case/definition.ts

```diff
@@ -2042,6 +2042,7 @@ export const enum State {
   CASE_SERVED = 'PREPARE_FOR_HEARING_CONDUCT_HEARING',
   CASE_WITHDRAWN = 'CASE_WITHDRAWN',
   CASE_DELETED = 'REQUESTED_FOR_DELETION',
+  JUDICIAL_REVIEW = 'JUDICIAL_REVIEW',
```

Comment:
I told you already please consider gate keeping as JR

Reply:
Yeah done this changes